Sources/Subs-BoardIndex.php

```diff
@@ -26,106 +26,113 @@
  * array with categories->boards->child_boards or an associative array
  * with boards->child_boards.
  *
- * @param array $boardIndexOptions An array of boardindex options
+ * @param array $board_index_options An array of boardindex options
  * @return array An array of information for displaying the boardindex
  */
 
-function getBoardIndex($boardIndexOptions)
+function getBoardIndex($board_index_options)
```

Comment:
curious as to what prompted this change

Reply:
Just a small analogy with the $message_index_selects in MessageIndex.php :)